### PR TITLE
Add redux-saga localStorage.

### PR DESCRIPTION
**Pull request by PennTx**
Use the `redux-saga` library to implement the `localStorage` feature.

### CODE REVIEW

File: src/features/rejection/add-question/add-question-hoc.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { connect } from 'react-redux';
```
Comment (ericelliott) [inline src/features/rejection/add-question/add-question-hoc.js]: I usually call the container component `index.js` inside the component/feature folder. `add-question-container.js` is also OK.

Comment (PennTx) [reply src/features/rejection/add-question/add-question-hoc.js]: Renamed to `index.js`.

File: src/features/rejection/rejection-sagas.spec.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      given: 'the loadLocalState saga',
+      should: 'first call the loadState function',
+      actual: gen.next().value,
+      expected: call(loadState)
+    });
+
+    assert({
+      given: 'the loadLocalState saga',
+      should: 'then put the loadQuestions function with the loaded state',
+      actual: gen.next().value,
```
Comment (ericelliott) [inline src/features/rejection/rejection-sagas.spec.js]: You don't need undefined if you pass an expected value into `gen.next(expectedValue)`

**Comment by ericelliott** [comment]
Much better!

File: src/features/rejection/rejection-sagas.spec.js (hunk 2 above)
Comment (PennTx) [reply src/features/rejection/rejection-sagas.spec.js]: Updated to input test state.

**Comment by PennTx** [comment]
Updated PR with requested changes.
Tests pass.
Merging.

**Pull request merged**